modules/module_manager/php/modulemanagerprovisioner.class.inc

```diff
@@ -0,0 +1,56 @@
+<?php declare(strict_types=1);
+/**
+ * This file implements a data provisioner to get all possible files
+ * for the media menu page.
+ *
+ * PHP Version 7
+ *
+ * @category   Behavioural
+ * @package    Main
+ * @subpackage Imaging
+ * @author     Henri Rabalais <henri.rabalais@mcin.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+
+namespace LORIS\module_manager;
+
+/**
+ * This class implements a data provisioner to get all possible files
+ * for the media menu page.
+ *
+ * PHP Version 7
+ *
+ * @category   Behavioural
+ * @package    Main
+ * @subpackage Imaging
+ * @author     Henri Rabalais <henri.rabalais@mcin.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+class ModuleManagerProvisioner extends \LORIS\Data\Provisioners\DBRowProvisioner
+{
+    /**
+     * Create a MediaFileProvisioner, which gets files for the meida
+     * menu table.
+     */
+    function __construct()
+    {
+        parent::__construct(
+            "SELECT name, active FROM modules",
+            array()
+        );
+    }
+
+    /**
+     * Returns an instance of a MediaFile object.
```

Comment:
```suggestion
     * Returns an instance of a ModuleRow object.
```

I'm not sure if this is precise enough of a suggestion but this should be updated.